web_programming/emails_from_url.py

```diff
@@ -0,0 +1,97 @@
+"""Get the site emails from URL."""
+__author__ = "Muhammad Umer Farooq"
+__license__ = "MIT"
+__version__ = "1.0.0"
+__maintainer__ = "Muhammad Umer Farooq"
+__email__ = "contact@muhammadumerfarooq.me"
+__status__ = "Production"
+
+from html.parser import HTMLParser
+import requests
+import re
```

Comment:
Please get in the habit of keeping your imports sorted (re before requests) so that when we have a lot of imports, we can quickly spot duplicate imports.